# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/base/nsContentUtils.h
# Commit: 98a751f6f41d
# Full Hash: 98a751f6f41d4e8d23f38aa0aa1d483f83cd0bda
# Author: Yulia Startsev <ystartsev@mozilla.com>
# Date: 2023-01-19 04:46:07
# Regressor Bug: 1247687
# File Overlap Count: 1
# Description:
#   Bug 1247687 - Implement csp for Module Workers; r=evilpie,asuth,rpl,ckerschb
#   
#   Depends on D155691
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D156102
# ==============================================================================

diff -r 78addb6af24d -r 98a751f6f41d dom/base/nsContentUtils.h
--- a/dom/base/nsContentUtils.h	Wed Jan 18 13:46:32 2023 +0000
+++ b/dom/base/nsContentUtils.h	Wed Jan 18 13:46:33 2023 +0000
@@ -3495,6 +3495,7 @@
     case nsIContentPolicy::TYPE_INTERNAL_SHARED_WORKER:
     case nsIContentPolicy::TYPE_INTERNAL_SERVICE_WORKER:
     case nsIContentPolicy::TYPE_INTERNAL_WORKER_IMPORT_SCRIPTS:
+    case nsIContentPolicy::TYPE_INTERNAL_WORKER_STATIC_MODULE:
     case nsIContentPolicy::TYPE_INTERNAL_AUDIOWORKLET:
     case nsIContentPolicy::TYPE_INTERNAL_PAINTWORKLET:
     case nsIContentPolicy::TYPE_INTERNAL_CHROMEUTILS_COMPILED_SCRIPT: